crates/ruff_python_formatter/src/statement/stmt_class_def.rs

```diff
@@ -128,15 +128,13 @@ impl FormatNodeRule<StmtClassDef> for FormatStmtClassDef {
         //
         // # comment
         // ```
-        empty_lines_before_trailing_comments(
-            comments.trailing(item),
-            if f.context().node_level() == NodeLevel::TopLevel {
-                2
-            } else {
-                1
-            },
-        )
-        .fmt(f)
+        let empty_lines = match (f.options().source_type(), f.context().node_level()) {
+            (PySourceType::Stub, NodeLevel::TopLevel) => 1,
+            (PySourceType::Stub, _) => 0,
+            (_, NodeLevel::TopLevel) => 2,
+            (_, _) => 1,
```

Comment:
it's a bit counterintuitive that the exceptions are the named cases

Reply:
Should `empty_lines` be inlined into `empty_lines_before_trailing_comments`. It seems the function is only used in `stmt_function_def` and `stmt_class_def` and both use the same implementation.